### PR TITLE
Upgrade script

### PR DESCRIPTION
**Pull request by ktalebian**
* Add `upgrade-plugin` script (pending unit test coverage)

### CODE REVIEW

File: src/prints/upgradePlugin.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import { Logger, singleLineString } from 'flex-plugins-utils-logger';
+
+const cracoUpgradeGuideLink = 'https://twilio.com';
```
Comment (ahcai) [inline src/prints/upgradePlugin.ts:3]: is this mean to just be a placeholder for now?

**Review by ahcai — APPROVED** [review]

File: src/__tests__/commands/flex/plugins/upgrade-plugin.test.ts [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  start()
+    .setup((instance) => {
+      sinon.stub(instance, 'pkg').get(() => ({
+        dependencies: { 'flex-plugin-scripts': '3.0.0' },
+      }));
+    })
+    .test((instance) => {
+      expect(instance.pkgVersion).to.equal(3);
+    })
+    .it('should should return then version of flex-plugin-scripts from dependencies');
```
Comment (ahcai) [inline src/__tests__/commands/flex/plugins/upgrade-plugin.test.ts:21]: a couple of your test descriptions have `should` twice 😛

File: src/__tests__/commands/flex/plugins/upgrade-plugin.test.ts [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+      expect(fs.writeJSONFile).to.have.been.calledOnce;
+      // @ts-ignore
+      const { match } = sinon;
+      expect(fs.writeJSONFile).to.have.been.calledWith(
+        match({ scripts: ctx.variables.untouchedScripts }),
+        match.string,
+        match.string,
+      );
+    })
+    .it('should remove packages that have no post/post');
```
Comment (ahcai) [inline src/__tests__/commands/flex/plugins/upgrade-plugin.test.ts:111]: what does no post/post mean?

File: src/prints/upgradePlugin.ts (hunk 1 above)
Comment (ktalebian) [reply src/prints/upgradePlugin.ts:3]: Ya, this is not published yet so I don't have a link for it.

File: src/__tests__/commands/flex/plugins/upgrade-plugin.test.ts (hunk 3 above)
Comment (ktalebian) [reply src/__tests__/commands/flex/plugins/upgrade-plugin.test.ts:111]: Typo :p did a replace all 😂

**Pull request merged**